article/_posts/2019-10-16-PickAndLearn.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: review
+title: "Pick-and-Learn: Automatic Quality Evaluation for Noisy-Labeled Image Segmentation"
+tags: deep-learning weakly-supervised segmentation
+author: "Nathan Painchaud"
+cite:
+    authors: "Haidong Zhu, Jialin Shi, Ji Wu"
+    title:   "Pick-and-Learn: Automatic Quality Evaluation for Noisy-Labeled Image Segmentation"
+    venue:   "MICCAI 2019"
+pdf: "https://arxiv.org/pdf/1907.11835.pdf"
+---
+
+
+# Highlights
+- Novel automatic quality evaluation module inspired by the relationship between labels and inputs
+- Overfitting control module to ensure enough trainable samples and avoid overfitting
+- Method can retain both high accuracy and good generalization at different noise levels
+
+
+# Introduction
+The motivation behind the paper is that it is still a struggle to get a massive amount of clean and precise labels to
+train a good model, and that low-accuracy segmentation model trained on noisy labels will severely impact the result.
+The authors also claim that up until now, most approaches for low-quality annotations are designed for the
+classification tasks.
+
+
+# Methods
+The overall proposed end-to-end architecure is detailed in Figure 2.  The authors used the dice loss as segmentation
+loss and U-Net as their segmentation network.
+
+![](/article/images/PickAndLearn/figure2.png)
+
+
+## Quality Awareness Module (QAM)
+The QAM runs parallel with the segmentation module to calculate weight values to apply to the loss from each sample.
+During training, the re-weighted loss is used for both the segmentation module and the QAM.
+
+The network used for the QAM is a VGG network, where the number of input channel is change to $$n + 1$$, where $$n$$ is
```

Comment:
the number of input channel : the number of input channels

Reply:
Typo corrected!

Reply:
Done in 6b69fdd.